src/shared/locales/ja/index.js

```diff
@@ -0,0 +1,21 @@
+import about from './about'
+import app from './app'
+// import edit from './edit'
```

Comment:
You should cancel all comments in this file. 
Reference here: https://github.com/agalwood/Motrix/blob/master/src/shared/locales/en-US/index.js